shells/pipes-shell-2/lib/utils.js

```diff
@@ -38,7 +39,9 @@ export const createPlanificator = async arc => {
     userid: 'user',
     storageKeyBase: 'volatile',
     onlyConsumer: false,
-    debug: true
+    //kill with fire
```

Comment:
is the comment intentional? 
i see a similar one below, so probably yes - can you rephrase it to be both more friendly and informative, please :)

Reply:
Haha, no, it's a marker to revisit before sending out. This obviously failed :-D will correct when at the computer.

Reply:
Removed the comment now, I hoped to remove the 'debug' flag, but didn't manage to do it yet.